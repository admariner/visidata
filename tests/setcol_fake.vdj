#!vd -p
{"row": "faker_salt", "longname": "set-option", "input": "1"}
{"longname": "open-file", "input": "sample_data/sample.tsv", "keystrokes": "o"}
{"sheet": "sample", "col": "", "row": "", "longname": "select-rows", "input": "", "keystrokes": "gs", "comment": "select all rows"}
{"sheet": "sample", "col": "Rep", "row": "", "longname": "setcol-fake", "input": "name", "comment": ""}
