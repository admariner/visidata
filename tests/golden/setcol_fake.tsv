OrderDate	Region	Rep	Item	Units	Unit_Cost	Total
2016-01-06	East	Paula Rogers	Pencil	95	1.99	189.05
2016-01-23	Central	Paul Lopez	Binder	50	19.99	999.50
2016-02-09	Central	Shawn Johnson	Pencil	36	4.99	179.64
2016-02-26	Central	Jeffery Gutierrez	Pen	27	19.99	539.73
2016-03-15	West	Eduardo Rodriguez	Pencil	56	2.99	167.44
2016-04-01	East	Paula Rogers	Binder	60	4.99	299.40
2016-04-18	Central	Joseph Cameron	Pencil	75	1.99	149.25
2016-05-05	Central	Shawn Johnson	Pencil	90	4.99	449.10
2016-05-22	West	Steven Mayer	Pencil	32	1.99	63.68
2016-06-08	East	Paula Rogers	Binder	60	8.99	539.40
2016-06-25	Central	Crystal Dyer	Pencil	90	4.99	449.10
2016-07-12	East	Kathryn Wood	Binder	29	1.99	57.71
2016-07-29	East	Robyn Massey	Binder	81	19.99	1619.19
2016-08-15	East	Paula Rogers	Pencil	35	4.99	174.65
2016-09-01	Central	William Beck	Desk	2	125.00	250.00
2016-09-18	East	Paula Rogers	Pen Set	16	15.99	255.84
2016-10-05	Central	Crystal Dyer	Binder	28	8.99	251.72
2016-10-22	East	Paula Rogers	Pen	64	8.99	575.36
2016-11-08	East	Robyn Massey	Pen	15	19.99	299.85
2016-11-25	Central	Paul Lopez	Pen Set	96	4.99	479.04
2016-12-12	Central	William Beck	Pencil	67	1.29	86.43
2016-12-29	East	Robyn Massey	Pen Set	74	15.99	1183.26
2017-01-15	Central	Jeffery Gutierrez	Binder	46	8.99	413.54
2017-02-01	Central	William Beck	Binder	87	15.00	1305.00
2017-02-18	East	Paula Rogers	Binder	4	4.99	19.96
2017-03-07	West	Eduardo Rodriguez	Binder	7	19.99	139.93
2017-03-24	Central	Shawn Johnson	Pen Set	50	4.99	249.50
2017-04-10	Central	Joseph Cameron	Pencil	66	1.99	131.34
2017-04-27	East	Kathryn Wood	Pen	96	4.99	479.04
2017-05-14	Central	Jeffery Gutierrez	Pencil	53	1.29	68.37
2017-05-31	Central	Jeffery Gutierrez	Binder	80	8.99	719.20
2017-06-17	Central	Paul Lopez	Desk	5	125.00	625.00
2017-07-04	East	Paula Rogers	Pen Set	62	4.99	309.38
2017-07-21	Central	Crystal Dyer	Pen Set	55	12.49	686.95
2017-08-07	Central	Paul Lopez	Pen Set	42	23.95	1005.90
2017-08-24	West	Eduardo Rodriguez	Desk	3	275.00	825.00
2017-09-10	Central	Jeffery Gutierrez	Pencil	7	1.29	9.03
2017-09-27	West	Eduardo Rodriguez	Pen	76	1.99	151.24
2017-10-14	West	Steven Mayer	Binder	57	19.99	1139.43
2017-10-31	Central	Joseph Cameron	Pencil	14	1.29	18.06
2017-11-17	Central	Shawn Johnson	Binder	11	4.99	54.89
2017-12-04	Central	Shawn Johnson	Binder	94	19.99	1879.06
2017-12-21	Central	Joseph Cameron	Binder	28	4.99	139.72
